pkgs/development/compilers/idris2/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, makeWrapper
+, clang, chez 
+}:
+
+# Uses scheme to bootstrap the build of idris2
+stdenv.mkDerivation {
+  name = "idris2";
+  version = "0.2.0-840e020";
+
+  nativeBuildInputs = [ makeWrapper ];
+  buildInputs = [ clang chez ];
+
+  prePatch = ''
+    patchShebangs --build tests
+  '';
+
+  makeFlags = "PREFIX=$(out)";
```

Comment:
```suggestion
  makeFlags = [ "PREFIX=$(out)" ];
```

Reply:
This is not relevant yet, but might be once we pass bash arrays rather than strings.

Reply:
Done. Thanks.